packages/Input/src/Input.js

```diff
@@ -85,7 +89,7 @@ const Input = props => {
         size={iconSize}
         onClick={inputClearHandler}
       >
-        <TimesCircleIcon />
+        {props.clearIcon}
```

Comment:
Is adding jsx components in default props considered a good practice? 

I'd prefer:

```js
{props.clearIcon || <TimesCircleIcon />} 
```

instead

Reply:
Do we ignore the default prop then?

Reply:
Also,

```js
const { hasClearButton, isDisabled, isReadOnly, size, value, clearIcon } = props;

...
{clearIcon || <TimesCircleIcon />}
```
instesd of 

```
props.clearIcon
```

Reply:
Okok, ill just set it to null

Reply:
> Do we ignore the default prop then?

I guess we still need a default prop `null`. 

Imagine you had a more complex scenario rather than just `<TimesCircleIcon />` without any props? It could be not possible to put it into default props at all anyway. 

Maybe, if we provide a default icon and allow user to customize it, then `customClearIcon` would be a more descriptive name than just `clearIcon`

Also, if we want to allow the user to have no icon, then 

```
{clearIcon === undefined || <TimesCircleIcon />}
```

that way you can do `<Input clearIcon={null} />`. 

@nahumzs cc

Reply:
we also have `hasClearButton` that needs to be set to true, so probably don't need that above case to set clearIcon to null.

Reply:
I can also rename it to `customClearIcon`, thoughts? @nahumzs